File: Baekjoon/Math/[2576]홀수/solution.py
import sys

res_min = 100
res_sum = 0

for i in range(7):
    num = int(sys.stdin.readline())

    if num % 2 != 0:
        res_min = num if res_min > num else res_min
        res_sum += num

if res_sum == 0:
    sys.stdout.write("-1\n")
else:
    sys.stdout.write(str(res_sum) + "\n" + str(res_min) + "\n")

File: Baekjoon/Math/[2576]홀수/solution.js
const fileName = process.platform === "linux" ? "/dev/stdin" : "example.dat";
const inputData = require("fs")
  .readFileSync(fileName)
  .toString()
  .trim()
  .split("\n");
let res_min = 100;
let res_sum = 0;

inputData.map((item) => {
  let num = Number(item);
  if (num % 2 != 0) {
    res_min = res_min > num ? num : res_min;
    res_sum += num;
  }
});

if (res_sum === 0) {
  console.log(-1);
} else {
  console.log([res_sum, res_min].join("\n"));
}
